ToDoApp/Extention/UITextViewExtention.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UITextViewExtention.swift
+//  ToDoApp
+//
+//  Created by 최용헌 on 2023/05/23.
+//
+
+import UIKit
+
+// MARK: - 문자 길이에 따라 크기 증가
+extension UITextView {
+  func adjustUITextViewHeight() {
+    self.translatesAutoresizingMaskIntoConstraints = true
+    self.sizeToFit()
+    self.isScrollEnabled = false
+  }
+  
+  // MARK: - 문자열 비었는지 확인
+  func nonEmpty(_ str: String) -> String? {
+    if str.isEmpty { return nil }
+    return str
+  }
```

Comment:
```suggestion
  func nonEmpty() -> String? {
    if self.isEmpty { return nil }
    return self.text
  }
```

이 부분 같은 경우는 `extension` 사용의 이점을 활용하지 못하고 있어.
`self`를 통해서 인스턴스의 프로퍼티에 접근할 수 있는 점을 활용해주도록 해줘!